src/Command/Initializer/Profiles/blank/Versions.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/**
+ * Slim Framework (https://slimframework.com)
+ *
+ * @license https://github.com/slimphp/Slim-Console/blob/0.x/LICENSE.md (MIT License)
+ */
+
+declare(strict_types=1);
+
+namespace Slim\Console\Command\Initializer\Profiles\blank;
+
+use Slim\Console\Command\Initializer\Profiles\Versions as ParentVersions;
+
+/**
+ * Defines version constants.
+ *
+ * @package Slim\Console\Command\Initializer\Profiles\blank
+ * @author Temuri Takalandze <me@abgeo.dev>
+ */
+class Versions extends ParentVersions
+{
+    public const GIZZLE_PSR_7 = '^1.6';
```

Comment:
I dont't like the fact, that we would hard code these versions in here. That means that we need to keep track of these repositories. Isn't there another solution? Dependabot wouldn't help us out.

Reply:
I have no other solutions at the moment.

Reply:
@adriansuter this is a hard problem to solve I suppose. Would there be a way to query composer perhaps to find latest version? The problem becomes that we must ensure all the deps are compatible and if we do this dynamically some issues may arise in the future. 

We can periodically update all the deps or we could create a manifest to pull versions from which the dependency objects could reference? I mean ultimately maybe we could require all those deps as dev deps of the console and look for their version in `composer.json`. Thoughts?

Reply:
There is something like an API provided by packagist, https://packagist.org/apidoc#get-package-data

But the problem is, that we can't blindly trust the newest versions. We should have some sort of unit testing for all the possible combinations of packages and versions, duh. If we stick with this method, then I fear, that sooner or later the slim console would use old versions.

Reply:
@adriansuter I think this API can provide the latest versions, but cannot check for dependency compatibility.

Reply:
@adriansuter I think this API can provide the latest versions, but cannot check for dependency compatibility.

Reply:
Why can’t we just manually update every so often instead? This way we have fixed versions that we know work. This is a problem we can fix later if it becomes too big of an issue.

Reply:
> This is a problem we can fix later if it becomes too big of an issue

Fine.